packages/terra-popup/src/PopupContent.scss

```diff
@@ -0,0 +1,54 @@
+@import '~terra-mixins';
+
+$terra-popup-margin: 9px;
+
+.terra-PopupContent-header {
+  box-shadow: 0 2px 2px 0 rgba(0, 0, 0, 0.15);
+  height: 30px;
+  width: 100%;
+}
+
+.terra-PopupContent-close {
+  background: 0 0;
+  border: 0;
+  color: #999;
+  cursor: pointer;
+  font-size: 1.45rem;
+  font-weight: 700;
+  line-height: 1;
+  padding: 0;
+
+  &:hover {
+    color: #111;
```

Comment:
It was just matching the current pattern for the modal dialog, I am still waiting on final ux.